### PR TITLE
Doc suggestion - preview_roots parameter in table_config.preview

### PR DESCRIPTION
**Pull request by xperrylinn**
# Citrine Python PR

## Description 
Suggesting to update the preview method preview_roots parameter docstring to say "material run" instead of "material history". It seems ambiguous in whether it could be a run, spec, or template. I was personally confused because the only available row definition takes in templates, not runs. Just a suggestion! :-)

### PR Type:
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

### Adherence to team decisions
- [ ] I have added tests for 100% coverage
- [ ] I have written Numpy-style docstrings for every method and class.
- [ ] I have communicated the downstream consequences of the PR to others.
- [ ] I have bumped the version in setup.py

### CODE REVIEW

File: src/citrine/resources/table_config.py [hunk 1]
```diff
@@ -383,7 +383,7 @@ def preview(self, table_config: TableConfig, preview_roots: List[LinkByUID]) ->
         table_config: TableConfig
             Table Config to preview
         preview_roots: List[LinkByUID]
-            List of links to the material history roots to use in the preview
+            List of links to the material runs to use in the preview
```
Comment (kroenlein) [inline src/citrine/resources/table_config.py:386]:
```suggestion
            List of links to the material runs to use as terminal materials in the preview
```

**Review by kroenlein — APPROVED** [review]
I think the suggestion makes it a little clearer what role the runs play, but maybe it's just adding unnecessary verbiage.

You need to bump the patch version in order to merge back to main branch -- this is true for every commit to the library, even if all edits are to documentation.

File: src/citrine/resources/table_config.py (hunk 1 above)
Comment (xperrylinn) [reply src/citrine/resources/table_config.py:386]: I dig it

**Comment by xperrylinn** [comment]
Bumped the version number and synced with the master branch.

**Review by kroenlein — APPROVED** [review]

**Pull request merged**